nltk/corpus/reader/mte.py

```diff
@@ -284,7 +284,7 @@ def paras(self, fileids=None):
 
     def lemma_words(self, fileids=None):
         """
-	    :param fileids: A list specifying the fileids that should be used.
```

Comment:
That's odd ... why would it remove `:`

Reply:
Wow, well spotted. There were a few occassions where black would incorrectly format as it would replace a tab with 8 spaces instead of 4. Or at least, that's what I assume the problem was. For these cases I manually fixed it. In this case, I must've accidentally removed the colon as well, and not realised.

Another common problem was:
```python
                self.random_code_here()
#              this is where we do random code
                self.more_random_code_here()
```
converting to
```python
                self.random_code_here()
                #              this is where we do random code
                self.more_random_code_here()
```

I manually changed these to
```python
                self.random_code_here()
                # this is where we do random code
                self.more_random_code_here()
```